include/circt/Dialect/FIRRTL/Passes.td

```diff
@@ -30,6 +30,16 @@ def LowerFIRRTLAnnotations : Pass<"firrtl-lower-annotations", "firrtl::CircuitOp
       "Ignore unknown annotations.">
   ];
   let dependentDialects = ["hw::HWDialect"];
+  let statistics = [
+    Statistic<"numRawAnnotations", "num-raw-annos",
+      "Number of raw annotations on circuit">,
+    Statistic<"numAddedAnnos", "num-added-annos",
+       "Number of additional annotations">,
+    Statistic<"numAnnos", "num-annos",
+       "Total number of annotations processed">,
```

Comment:
Thank you for implementing this exactly as I suggested. Thinking about it more, is it useful to break out `numAnnos` separately from `numRawAnnotations` and `numAddedAnnos` given that this is just the sum of the latter two? I'm fine with anything here.

Reply:
Hmm.

I'm fine either way, too.  Notes in its favor-- it's the least implementation-specific, for one, and adds information (beyond doing the math for us) in that it doesn't require knowledge that these are statistics of the same "kind" that do not overlap and that the sum is meaningful.  Thinking a bit about when I've found these useful or when I might: "Total" counters probably make more sense when there are more than 2 individual sub-counters, when sub-counters may overlap, or if the number of sub-counters may change over time --as a stable-ish metric (or one-stop number to check) robust to splitting counters into more granular stats, adding new types of handled things, altering pass behavior, etc...

I'm not convinced this is useful presently, thinking I'll just drop.  Not a big deal either way presently I don't think, but if we aren't sure it's useful then let's not have it until we are sure :smile:.  Easy to add in the future!